build-lib/lib.js

```diff
@@ -67,12 +67,14 @@ module.exports = function (pathConfig, env) {
     destDir: './node-lib'
   });
 
-  if (env.production) {
-    trees.push(uglifyJavaScript(funnel(trees, {
-      getDestinationPath: function (path) {
-        return path.replace(/\.js/, '.min.js');
-      }
-    })));
+  if (env == 'production') {
+    trees = [
```

Comment:
`uglifyJavaScript` apparently returns all the files in the fed in tree along with the minified version of the `js` files.

@naming, i had a bit of a dilemma there .... changing now.